docs/src/apimanual.md

```diff
@@ -510,6 +510,23 @@ See [`PositiveSemidefiniteConeTriangle`](@ref MathOptInterface.PositiveSemidefin
 
 [Explain `modifyconstraint!` and `modifyobjective!`.]
 
+### Constraint bridges
+
+A same constraint often possess different equivalent formulations but a solver may only support one of them.
+It would be duplicate work to implement rewritting rules in every solver wrapper for every different formulation of the constraint to express it in the form supported by the solver.
+Constraint bridges provide a way to define a rewritting rule to top of the MOI interface which can be used by any optimizer.
+Some rule also implement constraint modifications and constraint primal and duals translations.
+
+Given a bridge, the `@bridge` macro can be used to define a bridge optimizer that automatically apply the rewritting rule for a given optimizer.
```

Comment:
Maybe omit discussion of the bridge macro and focus on the pre-defined bridges. The macro should be an internal implementation detail that's only relevant if you're writing a new bridge.

Reply:
My intention in asking you to document how they're implemented was more about how they fit into the MOI framework rather than the specific `@bridge` macro.